frontend/ui/elements/screensaver_menu.lua

```diff
@@ -153,12 +154,24 @@ return {
                         end,
                     },
                     {
-                        text = _("Stretch covers and images to fit screen"),
+                        text_func = function()
+                            if G_reader_settings:nilOrFalse("screensaver_stretch_images") then
+                                return _("Stretch to fit screen: off")
+                            elseif G_reader_settings:readSetting("screensaver_stretch_limit_percentage") then
+                                return T(_("Stretch to fit screen: limit to %1%"),
+                                    G_reader_settings:readSetting("screensaver_stretch_limit_percentage"))
+                            else
+                                return _("Stretch to fit screen: full")
```

Comment:
I think you don't need to add noise to the 2 original settings which are obvious by themselves - you could just add some note when the feature you added is enabled. Taping on it will show you all there is to it.
```
[ ] Stretch to fit screen
[x] Stretch to fit screen
[x] Stretch to fit screen (with limit)
```

Reply:
OK.